contracts/FlowIDTableStaking.cdc

```diff
@@ -631,10 +615,13 @@ pub contract FlowIDTableStaking {
 
         /// Remove a node from the record
         pub fun removeNode(_ nodeID: String): @NodeRecord {
-            // Remove the node from the table
             let node <- FlowIDTableStaking.nodes.remove(key: nodeID)
                 ?? panic("Could not find a node with the specified ID")
 
+            FlowIDTableStaking.networkingAddressClaimed[node.networkingAddress] = nil
```

Comment:
Would `FlowIDTableStaking.networkingAddressClaimed.remove(key: node.networkingAddress)` work here? might be better regarding storage?

Reply:
Good catch! I also think calling remove makes it a bit more obvious that this removes an entry.
Semantics/performance wise the two variants are actually the same: Assigning nil calls remove internally: https://github.com/onflow/cadence/blob/13ee70e94308b3bb892c6907ec6c495ed3d85759/runtime/interpreter/value.go#L5885-L5894

Reply:
yeah I can switch to that. I forgot about that when writing it